ramls/items.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Since we use a "custom" item here, it make sense to me to put the whole API schema to a single file.

Reply:
Changed